python/nvstrings/cpp/pytext.cpp

```diff
@@ -649,6 +649,50 @@ static PyObject* n_scatter_count( PyObject* self, PyObject* args )
     return PyLong_FromVoidPtr((void*)strs);
 }
 
+//
+static PyObject* n_porter_stemmer_measure( PyObject* self, PyObject* args )
+{
+
+    PyObject* pystrs = PyTuple_GetItem(args,0);
+    NVStrings* strs = strings_from_object(pystrs);
+
+     if( strs==0 )
+        Py_RETURN_NONE;
```

Comment:
You could check this in the python code too and throw a `ValueError` perhaps.

Reply:
Added check , thanks for the review. 

https://github.com/rapidsai/cudf/blob/170e59be2442c079e56a950799c4c5e954712f33/python/nvstrings/nvtext.py#L379-L380